vscode-trace-extension/src/extension.ts

```diff
@@ -6,8 +6,12 @@ import { TraceExplorerAvailableViewsProvider } from './trace-explorer/available-
 import { TraceExplorerOpenedTracesViewProvider } from './trace-explorer/opened-traces/trace-explorer-opened-traces-webview-provider';
```

Comment:
A note, say for the future, the `Signed-off-by` footer (commit message, PR description) requires angle brackets around the email. E.g., `<ngondalia@blackberry.com>`.

And, speaking of commit messages, we usually format them [based on these guidelines](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html). So this could be for future contributions.